distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Aggregate_Column_Helper.enso

```diff
@@ -89,16 +89,30 @@ prepare_aggregate_columns aggregates table =
 
    Assumes that the `Aggregate_Column` is resolved. You may need to transform it
    with `resolve_aggregate` first.
-default_aggregate_column_name aggregate_column =
+default_aggregate_column_name aggregate_column include_column=True =
     case aggregate_column of
         Group_By c _ -> c.name
         Count _ -> "Count"
         Count_Distinct columns _ _ ->
             "Count Distinct " + (columns.map .name . join " ")
-        Percentile p c _ -> ((p*100).floor.to_text + "%-ile ") + c.name
+        Percentile p c _ -> ((p*100).floor.to_text + "%-ile ") + (if include_column then c.name else "")
         _ ->
             prefix = Meta.get_simple_type_name aggregate_column . replace "_" " "
-            prefix + " " + aggregate_column.column.name
+            c = aggregate_column.column
+            prefix + " " + (if include_column then c.name else "")
+
+## Utility function to check if all same column
```

Comment:
This comment doesn't say more than the method name itself, it's also unclear why `Count` can be ignored. Would be good to clarify in the doc comment what this method _really_ does.

Reply:
Will update the doc in the next task along with some tests for this.